src/formatDate.js

```diff
@@ -51,6 +51,38 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const fromFormatObj = {};
+  const fromF = fromFormat[3];
+  const toF = toFormat[3];
+  const dateArr = date.split(fromF);
+  const res = [];
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    fromFormatObj[fromFormat[i][0]] = dateArr[i];
+  }
+
+  if (fromFormat.find(el => el.startsWith('Y')).length === 2) {
+    if (fromFormatObj['Y'] > 20) {
+      fromFormatObj['Y'] = (+fromFormatObj['Y'] + 1900).toString();
+    } else {
+      fromFormatObj['Y'] = (+fromFormatObj['Y'] + 2000).toString();
+    }
```

Comment:
use ternary operator for that
```suggestion
     fromFormatObj['Y'] = fromFormatObj['Y'] > 20
        ? (+fromFormatObj['Y'] + 1900).toString()
        :  fromFormatObj['Y'] = (+fromFormatObj['Y'] + 2000).toString();
```